pygmt/src/grd2cpt.py

```diff
@@ -159,6 +161,14 @@ def grd2cpt(grid, **kwargs):
         range. Note that ``cyclic=True`` cannot be set together with
         ``categorical=True``.
     {verbose}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 30 arc minute grid with "gridline" registration
+    >>> grid = pygmt.datasets.load_earth_relief("30m", registration="gridline")
+    >>> # create a CPT with grd2cpt
+    >>> pygmt.grd2cpt(grid=grid)
```

Comment:
I still don't like this inline example, because it doesn't show how to use the CPT file. 
I think it's still necessary to call `Figure.grdimage()` to plot the grid.

Reply:
I added additional lines and comments to show how the CPT would be used.